tests/e2e-ibc/conformance_test.go

```diff
@@ -0,0 +1,59 @@
+package main_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"go.uber.org/zap/zaptest"
+
+	"github.com/strangelove-ventures/interchaintest/v7"
+	"github.com/strangelove-ventures/interchaintest/v7/conformance"
+	"github.com/strangelove-ventures/interchaintest/v7/ibc"
+	"github.com/strangelove-ventures/interchaintest/v7/testreporter"
+	"github.com/stretchr/testify/require"
+
+	kavainterchain "github.com/kava-labs/kava/tests/interchain"
+)
+
+var (
+	zero = 0
```

Comment:
These could be named based on what they are used for in addition to being constants?
```
const (
  numFullNodes = 0
  numValidators = 1
)
```

Reply:
yup yup haha they could and should 👍